app/views/posts/index.html.erb

```diff
@@ -0,0 +1,43 @@
+<p id="notice"><%= notice %></p>
```

Comment:
You installed `gem slim`, but you are using plain html.erb files.
If you have some time, using slim more preferable
Your template will be shorter.